pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -89,6 +90,35 @@ func (pl *VolumeBinding) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEvent {
+	events := []framework.ClusterEvent{
+		// Pods may fail because the storage class is not configured correctly.
+		// After the storage class is fixed (e.g. allowedTopologies,
```

Comment:
What's the exact meaning of "fixed", do you mean a storage class is updated to correct the configuration issue? If so, we can say sth. like:

```
Pods may fail because of missing or mis-configured storage class (e.g., allowedTopologies,
allowedTopologies), and hence may become schedulable upon StorageClass Add or Update events.
```

Reply:
updated